symengine/rational.h

```diff
@@ -236,9 +239,12 @@ void get_num_den(const Rational &rat,
         const Ptr<RCP<const Integer>> &den);
 
 //! convenience creator from two longs
-inline RCP<const Number> rational(long n, long d) {
-    return Rational::from_two_ints(n, d);
-    }
+inline RCP<const Rational> rational(long n, long d)
+{
+    mpq_class rat(n, d);
+    rat.canonicalize();
+    return make_rcp<const Rational>(rat);
+}
```

Comment:
What is the motivation of this replacement? The original code just called the following function:
```c++
RCP<const Number> Rational::from_two_ints(long n, long d)
{
    if (d == 0)
        throw std::runtime_error("Rational: Division by zero.");
    mpq_class q(n, d);

    // This is potentially slow, but has to be done, since 'n/d' might not be
    // in canonical form.
    q.canonicalize();

    return Rational::from_mpq(q);
}
```
Which does the same thing, except that one check for zero denominator.